src/main/webapp/i18n/de/team.json

```
@@ -68,9 +71,78 @@
+                    "items": {
+                        "conflictFreeTeam": {
+                            "label": "Konfliktfreies Team",
+                            "tooltip": "Dieses Team kann ohne Probleme importiert werden."
+                        },
+                        "teamShortNameConflict": {
+                            "label": "Team-Kurznamen Konflikt",
+                            "tooltip": "Ein Team mit dem gleichen Kurznamen existiert bereits in dieser Übung."
+                        },
+                        "studentConflict": {
+                            "label": "Studenten Konflikt",
+                            "tooltip": "Der rot markierte Student ist bereits in einem Team dieser Übung."
+                        }
+                    }
+                }
+            },
+            "loadSourceTeams": {
+                "failed": "Teams konnten nicht geladen werden. Bitte versuche es nochmal."
+            },
+            "noTeamsInSourceExercise": "Die Quell-Übung hat keine Teams.",
+            "importStrategy": {
+                "label": "Import-Strategie",
+                "tooltip": "Die Import-Strategie bestimmt, welche Quell-Teams importiert werden und was mit den existierenden Teams in dieser Übung geschieht.",
+                "options": {
+                    "purgeExisting": {
+                        "label": "Bestehende Teams löschen",
+                        "badge": "kritisch",
+                        "explanation": "Bevor alle Teams der Quell-Übung oben importiert werden, werden alle existierenden Teams dieser Übung inklusive ihrer Teilnahmen gelöscht."
+                    },
+                    "createOnly": {
+                        "label": "Nur Teams hinzufügen",
+                        "badge": "sicher",
+                        "explanation": "Nur konfliktfreie Quell-Teams (oben in grün angezeigt) werden importiert. Bereits bestehende Teams in dieser Übung werden nicht angefasst."
+                    }
+                }
+            },
+            "importPreview": {
+                "noTeamsToImport": "Keine Teams zum Importieren",
+                "noConflictFreeTeamsToImport": "Keine konfliktfreien Teams zum Importieren",
+                "teamsToBeDeleted": "Teams zum Löschen:",
+                "teamsToBeImported": "Teams zum Importieren:",
+                "outOf": "(von {{ number }})",
+                "totalTeamsAfterImport": "Teams insgesamt nach Import:"
+            },
+            "purgeConfirmationDialog": {
+                "question": "Sollen alle Teams der Übung <strong>{{ title }}</strong> und ihre Teilnahmen an der Aufgabe wirklich dauerhaft gelöscht werden? Diese Aktion kann NICHT rückgängig gemacht werden!",
+                "typeNameToConfirm": "Bitte gib den Titel der Übung zur Bestätigung ein."
+            },
+            "importError": "Import failed. Please try again or contact an administrator for assistance."
```

It seems like you forgot a translation here:

```suggestion
            "importError": "Import fehlgeschlagen. Bitte versuche es nochmal oder kontaktiere einen administrator für Hilfe."
```

---

Good catch, thanks!